src/app/app-routing.module.ts

```diff
@@ -8,13 +8,15 @@ import { Routes, RouterModule } from '@angular/router';
 import { LoginPageComponent } from './login-page/login-page.component';
 import { MainPageComponent } from './main-page/main-page.component';
 import { ProfilePageComponent } from './profile-page/profile-page.component';
-import { RecipeDetailPageComponent } from './recipe-detail-page/recipe-detail-page.component'
+import { RecipeDetailPageComponent } from './recipe-detail-page/recipe-detail-page.component';
+import { RecipeOverviewPageComponent } from './recipe-overview-page/recipe-overview-page.component'
 
 const routes: Routes = [
   {path: 'login', component: LoginPageComponent},
   {path: '', component: MainPageComponent},
   {path: 'profile', component: ProfilePageComponent},
   {path: 'recipe', component: RecipeDetailPageComponent},
+  {path: 'recipe/search', component: RecipeOverviewPageComponent},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7556402)
```suggestion
  {path: "recipe/search", component: RecipeOverviewPageComponent},
```